### DISCUSSION TITLE
Help translating ICSx⁵

### DISCUSSION THREAD

**Discussion by rfc2822** [bitfireAT/icsx5#33] [Announcements]
If ICSx⁵ is not already fully translated into your language, you could help!

All you need is a free Transifex account. Then you can submit translations here:

https://www.transifex.com/bitfireAT/icsx5/

**Comment by t1011** [comment]
Hello!
For quite some time now, I have done a translation into Russian. There have been two updates of the application, but the translation has not been updated. Please share any thoughts on this issue.

**Comment by rfc2822** [comment]
Hi,

Thanks for your hint. The Transifex client hasn't pulled German and Russian for some reason. I have now forced and and your translations should now be available: https://github.com/bitfireAT/icsx5/commit/f5b94975dd6c209cc5c77897ef985f549ef66901

Are these yours?

**Reply by t1011** [reply]
Yes, this is my translation. That's great!

**Comment by comradekingu** [comment]
Happy to keep my translations up to date if you move it over to Weblate in some capacity.
Full coverage doesn't always mean fully translated.

**Comment by guillaume-uH57J9** [comment]
Hi @rfc2822 ! I contributed to translating french strings. 74 out of 75 are now translated in Transifex, the only exception is the license term left in the original english form. 

Please consider pulling the translation from transifex to update the application. I noticed some strings I translated to french 1 year ago (eg "Synchronize now") do not appear to be translated in the app (ICSX5 2.1.1-standard installed from F-Droid).

Thanks for your work on ICXs5!

**Reply by ArnyminerZ** [reply]
@rfc2822 can you pull the translations?

**Reply by rfc2822** [reply]
@ArnyminerZ Can you call `scripts/fetch-translations.sh`? You should have necessary permissions.

**Comment by ezn24** [comment]
Hi, I’ve contributed the Traditional Chinese translation on Transifex, but I noticed it’s not included in the latest release.

**Reply by ArnyminerZ** [reply]
Hi, translations are periodically fetched. They will appear on the next release! Sorry for the delays 😓
